yogstation/code/modules/scripting/Interpreter/Interpreter.dm

```diff
@@ -95,6 +95,13 @@
 		CreateGlobalScope()
 			var/scope/S = new(program, null)
 			globalScope = S
+			for(var/datum/n_function/default/functype in subtypesof(/datum/n_function/default))
+				if(!istype(src, initial(functype.interp_type))
+					continue
```

Comment:
> yogstation/code/modules/scripting/Interpreter/Interpreter.dm: 100 :error: continue: missing comma ',' or right-paren ')'

Reply:
you're looking at unfinished code here okay?

Reply:
just realised that, sorry.

Reply:
Why would you commit code that doesn't run you doofus

Reply:
I was switching from my laptop to my PC okay?